ordem_producao/views.py

```
@@ -0,0 +1,223 @@
+    context = {
+        'ordem': ordem,
+        'cartas_qualidade': cartas_qualidade,
+        'os_manutencao': os_manutencao,
+        'os_qualidade': os_qualidade,
+        **stats,  # Unpack stats dict
+    }
+    return render(request, 'ordem_producao/ordem_producao_detail.html', context)
+
+
+@login_required
+def ordem_producao_create(request):
```

Pylint (3.3.7)

[refactor] Too many local variables (22/15)

(R0914)